src/test/java/Pages/AddUserPage.java

```diff
@@ -0,0 +1,36 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+
+public class AddUserPage {
+  WebDriver driver;
+
+  public AddUserPage(WebDriver driver){
+    this.driver=driver;
+  }
+
+  By nameField = By.id("name");
```

Comment:
I suggest reading about PageFactory: https://www.browserstack.com/guide/page-object-model-in-selenium to make page elements more readable.